src/DataFrames.jl

```diff
@@ -50,9 +52,7 @@ export AbstractDataFrame,
        unique!,
        unstack
 
-if VERSION >= v"1.1.0-DEV.792"
-    import Base.eachcol, Base.eachrow
-else
+if VERSION < v"1.1.0-DEV.792"
```

Comment:
Two questions as I do not know Compat.jl well enough:
* do we have to `export` something we import via `Compat.`? (probably yes, but I just wanted to make sure)
* shoud `import Compat.` be used even on current versions of Julia or it should be in the `if` block here.

Reply:
> * do we have to `export` something we import via `Compat.`?

My understanding is that `Compat` is "just" a Julia package.  So, if you want `eachrow` to work without explicitly importing `Compat`, I think you need to re-export `eachrow` from DataFrames.jl.

> * shoud `import Compat.` be used even on current versions of Julia or it should be in the `if` block here.

I don't think it matters since `Base.eachrow === Compat.eachrow` for `julia` >= 1.1 and `julia` does not care how `eachrow` is imported.  But my guess is that it is rather an implementation detail of `julia` so

```julia
if VERSION < v"1.1.0-DEV.792"
    import Base: eachcol, eachrow
else
    import Compat: eachcol, eachrow
end
```

makes sense if you want to play on the very safe side.

Reply:
I pushed the change I just suggested 9f1d862285b9399a42f775efda0fd5e4739b2f07.  Let me know if you want to revert it.